code/game/gamemodes/clock_cult/clock_ratvar.dm

```diff
@@ -120,31 +124,59 @@
 				sleep(125)
 				make_glow()
 				animate(glow, transform = matrix() * 3, alpha = 0, time = 5)
-				sleep(5)
-				new/obj/structure/clockwork/massive/ratvar(get_turf(src))
-				qdel(src)
+				var/turf/startpoint = get_turf(src)
+				sleep(3)
+				QDEL_IN(src, 3)
+				clockwork_gateway_activated = TRUE
+				if(ratvar_portal)
+					new/obj/structure/clockwork/massive/ratvar(startpoint)
+				else
+					var/message = text2ratvar("Behold")
+					world << "<span class='ratvar'>\"[message]!\"</span>"
+					world << "<span class='inathneq_large'>\"[message]!\"</span>"
+					world << "<span class='sevtug_large'>\"[message]!\"</span>"
+					world << "<span class='nezbere_large'>\"[message]!\"</span>"
+					world << "<span class='nzcrentr_large'>\"[message]!\"</span>"
+					world << 'sound/magic/clockwork/invoke_general.ogg'
+					var/x0 = startpoint.x
+					var/y0 = startpoint.y
+					for(var/I in spiral_range_turfs(255, startpoint))
```

Comment:
This loop isn't particularly cheap, but it is a lot cheaper than an equally-sized explosion, and it's not 'not cheap' for the reason you might assume looking at it:
It makes a lot of pooled overlays, which get deleted via a timer, and mostly slows the timer subsystem down.